docs/build/js-client/4-call-functions.md

```diff
@@ -0,0 +1,67 @@
+# Call Aqua functions
+
+# TODO: adapt to JS Client
+
+# TODO: mention aqua arrays and their types convertion
+
+## Bullets
+
+- write aqua
+- export function
+- compile
+- call from code
+
+## Understanding the compiler output
+
+For every exported function definition in aqua the compiler generated two overloads. One accepting the `FluencePeer` instance as the first argument, and one without it. Otherwise arguments are the same and correspond to the arguments of aqua functions. The last argument is always an optional config object with the following properties:
+
+- `ttl`: Optional parameter which specify TTL (time to live) of particle with execution logic for the function
```

Comment:
1. Why a list is introduced if it has the only member (`ttl`)?
 2. The previous sentence ends with "the following properties:" but just an optional parameter is described. How comes?

Reply:
Thanks for you comment. I think we have to postpone these section to after the Devner event